hs_core/templates/pages/genericresource.html

```diff
@@ -460,6 +459,29 @@
                             </form>
                         {% endif %}
                     </div>
+
+                    {% if is_owner_user %}
+                        <div class="clear-fix" id="shareable-container">
+                            {#   Shareable flag    #}
+                            <form class="pull-left" action="/hsapi/_internal/{{ cm.short_id }}/change-permissions/"
+                                  method="POST">
+                                {% csrf_token %}
+                                {% if cm.raccess.shareable %}
+                                    <input name="t" type="hidden" value="make_not_shareable"/>
+                                    <div class="checkbox" data-toggle="tooltip"
+                                         title="When checked, users other than owners may grant access to the resource.">
+                                        <label><input id="btn-shareable" type="checkbox" checked>Shareable</label>
+                                    </div>
+                                {% else %}
+                                    <input name="t" type="hidden" value="make_shareable"/>
+                                    <div class="checkbox" data-toggle="tooltip"
+                                         title="When checked, users other than owners may grant access to the resource.">
+                                        <label><input id="btn-shareable" type="checkbox">Shareable</label>
+                                    </div>
+                                {% endif %}
```

Comment:
@pkdash I am a bit confused with the if...else logic here. The only difference between if part and else part is the hidden field value. Is there a way to consolidate this if..else logic?